hack/build/docker/builder/Dockerfile

```diff
@@ -1,5 +1,10 @@
 FROM fedora:29
 
+RUN dnf install -y autoconf automake libtool python
```

Comment:
all the dnf stuff on one line would be ideal

Reply:
done

Reply:
done